web/client/deploy.sh

```diff
@@ -9,23 +9,22 @@ ts="$(date +%s)"
 dest="$rootdir/assets/$ts"
 echo "Deploying to $dest .."
 
-jspm install
-jspm bundle -m src/main.js
+npm install
+webpack
 
 mkdir -p "${dest}"
 
 tmpdest="$(mktemp -d)"
 
 # "content" is currently using an absolute directory, so don't copy it.
-cp -R index.html src/ jspm_packages/ jspm.config.js jspm.browser.js build.js build.js.map "${tmpdest}"
+cp -R index.html bundle.js bundle.js.map favicon.ico favicon-152.png "${tmpdest}"
```

Comment:
are favicon.ico and favicon-152.png correct? I don't recall seeing them in this directory.